client/settings/reporting-settings/components/export-language/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { SelectControl, ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import interpolateComponents from '@automattic/interpolate-components';
+import { ReportingExportLanguageHook } from '../../interfaces';
+import { useReportingExportLanguage } from 'wcpay/data';
+
+const ExportLanguage: React.FC = () => {
```

Comment:
Random thought: The code here looks a bit like the language selector in the modal component. I was wondering if it's possible to reuse this component in the modal?

Reply:
My initial plan was to do that, but it would require adding a good amount of extra code to reuse the component, which isn't practical. Instead, I've exported the options variable as a reusable constant that both components can share.